src/Stryker.Core/Stryker.Core.UnitTest/Options/Inputs/BasePathInputTests.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Stryker.Core.UnitTest.Options.Inputs
+{
+    class BasePathInputTests
```

Comment:
Missing yet? Or just something forgotten to be removed? :)

Reply:
I think @richardwerkman created the test classes for all time new inputs at once and is slowly filling them but good to point out so we don't miss it